<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Digital-7 Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="rpm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>190</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="rpm_value">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>390</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7 Mono</family>
      <pointsize>105</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>22000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rpm_unit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>385</width>
      <height>60</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Digital-7 Mono</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="speed">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>0</y>
     <width>410</width>
     <height>190</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="speed_value">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>230</width>
      <height>140</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7 Mono</family>
      <pointsize>105</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>220</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speed_unit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>225</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7 Mono</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KMPH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>204</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="gear_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>204</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital-7 Mono</family>
      <pointsize>230</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>410</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>410</width>
      <height>410</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="water_temp_info">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>water</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="water_temp_value">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>38</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>100</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="oil_temp_info">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Oil</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="oil_temp_value">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>38</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="intake_temp_info">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Intake</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="intake_temp_value">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>38</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="break_balance_info1">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B B</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="break_balance_value">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>38</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="info1_info">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>test</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="info1_value">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>LEMON MILK</family>
           <pointsize>38</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>190</y>
     <width>205</width>
     <height>230</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="TCS_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>205</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LEMON MILK</family>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TCS_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>165</y>
      <width>205</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LEMON MILK</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TCS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>614</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="warning_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>614</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LEMON MILK</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>190</y>
     <width>205</width>
     <height>230</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="test_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>205</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LEMON MILK</family>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="test_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>165</y>
      <width>205</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LEMON MILK</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
